exla/lib/exla/mlir/value.ex

```diff
@@ -66,19 +66,16 @@ defmodule EXLA.MLIR.Value do
     comparison_type =
       cond do
         Nx.Type.complex?(lhs_type) or Nx.Type.complex?(rhs_type) ->
-          attr_comparison_type(:float)
+          [compare_type: attr_comparison_type(:float)]
 
         Nx.Type.float?(lhs_type) or Nx.Type.float?(rhs_type) ->
-          attr_comparison_type(:float)
+          [compare_type: attr_comparison_type(:float)]
 
         true ->
-          attr_comparison_type(:notype)
```

Comment:
Let's also remove it here:

https://github.com/elixir-nx/nx/blob/08dc89336c5b15c42c2132b44f5aa28f93fd13ee/exla/lib/exla/mlir/value.ex#L1072